diff --git a/buyer/models/o_auth_response.go b/buyer/models/o_auth_response.go
index abe256d..b020c8d 100644
--- a/buyer/models/o_auth_response.go
+++ b/buyer/models/o_auth_response.go
@@ -24,8 +24,9 @@ type OAuthResponse struct {
 	// refresh token timeout
 	RefreshTokenTimeout string `json:"refresh_token_timeout,omitempty"`
 
-	// token timeout
-	TokenTimeout int64 `json:"token_timeout,omitempty"`
+        // token timeout
+        // TODO: Fix go-swagger, add string to int64 timeout
+        TokenTimeout int64 `json:"token_timeout,string,omitempty"`
 
 	// token type
 	TokenType string `json:"token_type,omitempty"`
